rsts/deployment/deployment/multicluster.rst

```diff
@@ -83,24 +82,24 @@ Some Flyte deployments may choose to run the control plane separate from the dat
 plane. FlyteAdmin is designed to create Kubernetes resources in one or more
 Flyte data plane clusters. For the admin to access remote clusters, it needs
 credentials to each cluster.
+Flyte makes use of Kubernetess Service Accounts to enable every data plane cluster to perform
+authenticated requests to the K8s API Server.
+The default behaviour is that ``FlyteAdmin`` creates a `ServiceAccount <https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/templates/admin/rbac.yaml#L4>`_
+in each data plane cluster. 
+In order to verify requests, the API Server expects a `signed bearer token <https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens>`__
+attached to the Service Account. 
 
-In Kubernetes, scoped service credentials are created by configuring a "Role"
-resource in a Kubernetes cluster. When you attach the role to a "ServiceAccount",
-Kubernetes generates a bearer token that permits access. Hence, create a
-FlyteAdmin `ServiceAccount <https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/templates/admin/rbac.yaml#L4>`_
-in each data plane cluster to generate these tokens.
 
-.. warning::
-  
-   **Never delete a ServiceAccount 🛑**
-
-   When you first create the FlyteAdmin ``ServiceAccount`` in a new cluster, a
-   bearer token is generated and will continue to allow access unless the
-   "ServiceAccount" is deleted.
+.. note::
+   As of Kubernetes 1.24 an above, the bearer token has to be generated manually for a Service Account, using the following command:
 
-To feed the credentials to FlyteAdmin, you must retrieve them from your new data plane cluster and upload them to admin (for example, within Lyft, `Confidant <https://github.com/lyft/confidant>`__ is used).
+   .. prompt:: bash $
+   
+     kubectl create token <service-account-name> -n <namespace>
+  
+To feed the credentials to FlyteAdmin, you must retrieve them from your new data plane cluster and upload them to ``FlyteAmin``.
```

Comment:
I would not use the word "upload" which might be misleading. I'd say that you need update your control plane configuration with the new secrets